### PR TITLE
sis-local: Update install path and improve running_in_sis() impl

### PR DESCRIPTION
**Pull request by sfc-gh-jcarroll**
- Switch install path to the new repo
- Improve `running_in_sis()` example function to be more reliable (using fix provided by @sfc-gh-bhess)

cc @sfc-gh-zblackwood FYI

### CODE REVIEW

File: samples/streamlit-in-snowflake/sis-local/README.md [hunk 1]
```diff
@@ -33,11 +33,9 @@ from snowflake.snowpark.exceptions import SnowparkSessionException
...
 
 def running_in_sis() -> bool:
-    try:
-        sess = get_active_session()
-        return True
-    except SnowparkSessionException:
-        return False
+    import snowflake.connector.connection
+    import inspect
+    return 1 == len([x for x in inspect.getmembers(snowflake.connector.connection) if x[0] == 'StoredProcConnection'])
```
Comment (sfc-gh-zblackwood) [inline samples/streamlit-in-snowflake/sis-local/README.md:38]: Sweet. I assume this is much faster than trying `get_active_session()`. Would you mind adding a comment to explain this? Why does `snowflake.connector.connection.StoredProcConnection` existing mean it's in SiS?

Comment (sfc-gh-jcarroll) [reply samples/streamlit-in-snowflake/sis-local/README.md]: Added a comment and switched it to the inverse (`SnowflakeConnection` exists externally, `StoredProcConnection` exists internally).

Apparently there are (rare) situations where `get_active_session()` will return true outside of SiS as well. So this is faster and more consistent.

**Review by sfc-gh-zblackwood — APPROVED** [review]
LGTM

**Comment by sfc-gh-jcarroll** [comment]
@iamontheinet would you be able to review and merge if looks good?

**Pull request merged**